projects/plugins/boost/app/assets/src/js/modules/image-size-analysis/RecommendationsMeta.svelte

```diff
@@ -122,7 +127,15 @@
 	<!-- Show a button to view the report if it's in progress or completed. -->
 	{#if [ ISAStatus.Queued, ISAStatus.Completed ].includes( $isaSummary.status ) && ! requestingReport}
 		<div class="button-area">
-			<Button href="#image-size-analysis/all/1" disabled={requestingReport}>
+			<Button
+				disabled={requestingReport}
+				on:click={() =>
+					recordBoostEventAndRedirect(
+						'#image-size-analysis/all/1',
+						'clicked_view_isa_report',
```

Comment:
Maybe this should be more specific to what it's tied to? If we were to add another place to check reports from, this name wouldn't be unique enough.

Reply:
Changed that to "clicked_view_isa_report_on_summary_page"